sample/README.md

```diff
@@ -34,4 +34,5 @@ If you recently issued new credentials, try re-installing the sample app as the
 Confirm the following:
 1. The proper package name (e.g. `com.example.buttonkit`) is shown in the mParticle Dashboard.
 2. There exists an input (the sample/test app) and an output (Button integration from mParticle marketplace) and that it is active.
-3. The Button integration is configured properly with a valid [Application ID](https://app.usebutton.com/account/login/?next=/settings/organization)
+3. The Button integration is configured properly with a valid [Application ID](https://app.usebutton.com/account/login/?next=/settings/organization).
```

Comment:
Can we specify _where_ the Button App id is configured (i.e. mParticle dashboard)